stats/justfile

```diff
@@ -26,7 +26,7 @@ test *args:
 
 test-with-db *args:
     -just db-port="{{test-db-port}}" db-name="" docker-name="{{docker-name}}-test" start-postgres
-    just db-port="{{test-db-port}}" db-name=""                                    test {{args}}
+    -just db-port="{{test-db-port}}" db-name=""                                    test {{args}}
```

Comment:
so you want to remove test table if test failed? 
as an example tests in `sqlx` remove test data only if test succeed:
https://docs.rs/sqlx/latest/sqlx/attr.test.html#automatic-test-database-management-requires-migrate-feature

```
Test databases are automatically cleaned up as tests succeed, but failed tests will leave their databases in-place to facilitate debugging. Note that to simplify the implementation, panics are always considered to be failures, even for #[should_panic] tests.
```